src/main/follower.ts

```diff
@@ -10,37 +10,32 @@ import type { IpcMainInvokeEvent } from 'electron';
 import logger from 'electron-log';
 import { SYNTHETIX_IPNS } from '../const';
 import { ROOT } from './settings';
-import { getPid, getPidsSync } from './pid';
+import { getPid, PID_FOLLOWER_FILE_PATH } from './pid';
 import unzipper from 'unzipper';
 import { getPlatformDetails } from './util';
+// import { rpcRequest } from './ipfs';
 
 // Change if we ever want to store all follower info in a custom folder
 const HOME = os.homedir();
 const IPFS_FOLLOW_PATH = path.join(HOME, '.ipfs-cluster-follow');
 
 export function followerKill() {
   try {
-    getPidsSync(
-      process.platform === 'win32'
-        ? 'ipfs-cluster-follow.exe'
-        : '.synthetix/ipfs-cluster-follow/ipfs-cluster-follow synthetix run'
-    ).forEach((pid) => {
-      logger.log('Killing ipfs-cluster-follow', pid);
-      process.kill(pid);
-    });
-  } catch (_e) {
-    // whatever
+    const pid = readFileSync(PID_FOLLOWER_FILE_PATH, 'utf8');
+    if (pid) {
+      rmSync(PID_FOLLOWER_FILE_PATH);
+      if (process.platform === 'win32') {
+        execSync(`taskkill /F /PID ${pid}`);
+      } else {
+        process.kill(Number(pid));
+      }
+      logger.log(`follower kill: PID ${pid} killed and PID file removed`);
+    }
+  } catch (e) {
+    logger.log('follower kill error:', e);
```

Comment:
`logger.error(e)`

But logging is not working during shutdown so probably not much sense to have it at all

Reply:
removed